packages/now-cli/src/commands/whoami.js

```diff
@@ -52,9 +52,17 @@ const main = async ctx => {
   }
 
   const debug = argv['--debug'];
-  const { authConfig: { token }, apiUrl } = ctx;
+  const {
+    authConfig: { token },
+    apiUrl,
+  } = ctx;
   const output = createOutput({ debug });
-  const client = new Client({ apiUrl, token, debug });
+  const client = new Client({
+    apiUrl,
+    token,
+    debug,
+    currentTeam: ctx.config.currentTeam,
```

Comment:
Tackled in 7dcf0cc. I've chosen to show both in that format:
```
> <username> (team: <teamname>)
```
Is that an acceptable compromise in your opinion?

Reply:
This should be fine, but I'd consider it a breaking change, because if someone used the output of `now whoami` to compare it to something they now get a different result.

Reply:
100% agree. I've considered outputting the team to `stderr` to avoid this breaking change. Wdyt?

Reply:
@lucleray yes, I think that would also work